Libplanet.Tests/Store/BlockSetTest.cs

```diff
@@ -92,7 +92,11 @@ public void CanRemoveItem()
             Assert.False(_set.Remove(_fx.Block1.Hash));
             _set[_fx.Block1.Hash] = _fx.Block1;
             Assert.True(_set.Remove(_fx.Block1.Hash));
-            Assert.Throws<KeyNotFoundException>(() => { var val = _set[_fx.Block1.Hash]; });
+            Assert.Throws<KeyNotFoundException>(() =>
+            {
+                var val = _set[_fx.Block1.Hash];
+                Console.Write(val);
```

Comment:
```suggestion
                var _ = _set[_fx.Block1.Hash];
```

As many cases, it also may need to use `#pragma` for stylecop. 😇